src/modules/commander/commander_params.c

```diff
@@ -416,6 +416,17 @@ PARAM_DEFINE_FLOAT(COM_OF_LOSS_T, 1.0f);
  */
 PARAM_DEFINE_INT32(COM_OBL_ACT, 0);
 
+/**
+ * Set command after a quadchute
+ *
+ * @value -1 No action: stay in current flight mode
+ * @value  0 Return mode
+ * @value  1 Land mode
+ * @value  2 Hold mode
+ * @group Commander
+ */
+PARAM_DEFINE_INT32(COM_QC_ACT, 0);
```

Comment:
Probably it would be worth mentioning that this is VTOL feature like also the naming in https://github.com/PX4/PX4-Autopilot/commit/05204b07a8525f6340cf89daebbdf8c8403fdff8#diff-98b4e0f87126f3486c8dda8f693ee45c682a856f17958aed5a33d51e5fd27766R1130
```suggestion
 * Action to take after a VTOL quadchute
 *
 * @value -1 No action: stay in current flight mode
 * @value  0 Return mode
 * @value  1 Land mode
 * @value  2 Hold mode
 * @group Commander
 */
PARAM_DEFINE_INT32(COM_VT_QC_ACT, 0);
```

What do you think @sfuhrer ?

Reply:
yeah agree, we don't need the _VT for it :+1: